### DISCUSSION TITLE
Can add Japanese (ja-JP) support to Moziila.Firefox?

### DISCUSSION THREAD

**Discussion by FX8350** [microsoft/winget-pkgs#37403] [Q&A] [answered]
Hello!(Konnichiwa!)

I tried to install Firefox using winget. Then the following error message is displayed
`インストーラーのハッシュが一致しません; このチェックを無効にするには、--force を使用します`
https://github.com/microsoft/winget-cli/blob/4968df49d5ee25565fa9c37719e976c06787d97a/Localization/Resources/ja-JP/winget.resw#L276-L277

I think this is happening because there is no Japanese language support in the [package](https://github.com/microsoft/winget-pkgs/tree/master/manifests/m/Mozilla/Firefox/95.0).

I found a similar discussion, it said that the language would not be added without requests from the user.
https://github.com/microsoft/winget-pkgs/discussions/35387

Would you please add Japanese language support?

**Comment by OfficialEsco** [comment, marked as answer]
`Installer Hash Mismatch Override Required` ?
I've added the ja-JP installer in https://github.com/microsoft/winget-pkgs/pull/37417, however you are linking to hash missmatch, so i guess ill check the hashes at the same time..

edit: it seems like the issue was the msix

**Reply by FX8350** [reply]
Maybe this, `Installer hash does not match; to override this check use --force`

Thanks for your help! Now I can install without `--force` command.

Arigatou!
